packages/client/src/common/hooks/useGetLocations.hook.ts

```diff
@@ -32,6 +32,7 @@ export const useGetLocations = (id?: number) => {
 				const locations = await trainingClient.get<LocationResponse>(`locations/${id}`);
 				if (locations.data.status === 200) {
 					setRowData(locations.data.data);
+					setLocationData(locations.data.data);
```

Comment:
What's the difference between rowData and locationData if they both get the same object?